srcpkgs/scanmem/template

```diff
@@ -14,7 +14,9 @@ distfiles="https://github.com/${pkgname}/${pkgname}/archive/v${version}.tar.gz"
 checksum=f02054b91322cf41517506158fcb74554e9fc6644e696f8aa25e5acf162d374b
 CFLAGS="-D__NEED_pid_t"
 
-nocross="checking for /proc/self/maps... configure: error"
+if [ "$CROSS_BUILD" ]; then
+	configure_args+=" --disable-procmem"
+fi
```

Comment:
Sorry but I couldn't follow. `/proc/<pid>/mem` is available on all our targets.
I think the correct approach would be add an `AC_CACHE_CHECK` for some value instead https://www.gnu.org/software/autoconf/manual/autoconf-2.67/html_node/Caching-Results.html